Update FAQ with changing transport guide. (#2539)
vis2k/Mirror: doc/Articles/FAQ.md

- Socket connections, includes people without spawned prefabs, non authenticated, or that may have bugged out but during spawn, but have connected temporarily (Android users minimising game).  Only host / server can check this.

NetworkManager.singleton.numPlayers

> - Number of active spawned player objects across all connections on the server (only the host / server can check this)

Reviewer: ```suggestion
- Number of active spawned player objects on the server (only the host / server can check this)
```
There's only one player object per connection, so that phrase is redundant.

</details>

### Changing Transports

<details>
   <summary>Changing transports is very easy and only requires a few steps</summary>

> - Go to the GameObject that has a NetworkManager attached

Reviewer: ```suggestion
- Go to the game object that has the `NetworkManager` component.
```

> - Remove the current transport script
> - Add another transport script via the Add Component button
> - Drag the transport script to the "Transport" field in the NetworkManager

Reviewer: ```suggestion
- Add another transport script via the Add Component button
- Drag the new transport script to the "Transport" field in the `NetworkManager`
- Remove the current transport script (optional)
```
Re-ordered steps because Network Manager may not allow removal of the only transport, or will auto-add it again.

> You have successfully changed transports. If you have connection issues with a transport that requires portforwarding, make sure to portforward the correct protocol (TCP/UDP).

Reviewer: ```suggestion
You have successfully changed transports. If you have connection issues with a transport that requires port forwarding, make sure to port forward the correct protocol (TCP / UDP).
```

> NOTE: Some transports are not included with Mirror and need to be downloaded from their resepective GitHub pages.

Reviewer: ```suggestion
NOTE: Some transports are not included with Mirror and need to be downloaded from their respective GitHub pages.
```
